libdnf/rpm/package.cpp

```diff
@@ -102,6 +102,10 @@ std::string Package::get_full_nevra() const {
     return cstring2string(solv::get_full_nevra(pool, id.id));
 }
 
+std::string Package::get_na() const {
+    return get_name() + "." + get_arch();
```

Comment:
It is not the the fastest approach how to create the string. 'Append' operator is faster then `+`.  Also get_arch() generates a temporal string that is deleted.

Reply:
Fixed.